Artifacts/windows-slack/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Slack",
+  "description": "Installs Slack using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Slack"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/slack.1.0.0.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
Initially, I just wanted to use  chocolatey package "slack". But  this package uses a custom installer which does not work in silent mode.. Installation of the artifact hung during the creation of the machine. But I found a standart msi installer And I create a new package (on myget feed [not chocolatey.org]).